sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/command/AlterTableRenamePartitionSuite.scala

```diff
@@ -25,4 +25,20 @@ import org.apache.spark.sql.execution.command.v1
  */
 class AlterTableRenamePartitionSuite
   extends v1.AlterTableRenamePartitionSuiteBase
-  with CommandSuiteBase
+  with CommandSuiteBase {
+
+  test("hive client calls") {
+    withNamespaceAndTable("ns", "tbl") { t =>
+      sql(s"CREATE TABLE $t (id int, part int) $defaultUsing PARTITIONED BY (part)")
+      sql(s"INSERT INTO $t PARTITION (part=0) SELECT 0")
+
+      checkHiveClientCalls(expected = 21) {
```

Comment:
The number is huge because there are a lot of checks like the same table exists (repeats many times) and loading of the table/its metadata from Hive catalog.